pixeltable/env.py

```diff
@@ -369,7 +400,10 @@ def _set_up(self, echo: bool = False, reinit_db: bool = False) -> None:
                     self._logger.error(f'Invalid time zone specified in configuration: {tz_name}')
 
         if reinit_db and self._store_db_exists():
-            self._drop_store_db()
+            if db_connect_str is not None:
+                self._delete_all_tables()
```

Comment:
Again, there are assumptions here that aren't spelled out. Probably best to do in the class comment.

Also: this seems *extremely* dangerous in the cloud scenario.

Reply:
we will not delete any tables or db in cloud case 
added a check for reinit_db can not be set when in cloud setup